ocp_resources/namespace.py

```diff
@@ -1,6 +1,5 @@
 from ocp_resources.logger import get_logger
-from ocp_resources.resource import Resource
-
+from ocp_resources.resource import Resource, TIMEOUT
```

Comment:
TIMEOUT is not needed.

Reply:
I want to use it instead of '240' so that the two will be bonded.

Reply:
ack.